x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/norms_parameter.tsx

```diff
@@ -19,7 +19,8 @@ export const NormsParameter = ({ configPath = 'norms' }: { configPath?: NormsPar
       defaultMessage: 'Use norms',
     })}
     description={i18n.translate('xpack.idxMgmt.mappingsEditor.useNormsFieldDescription', {
-      defaultMessage: 'Whether field-length should be taken into account when scoring queries.',
+      defaultMessage:
+        'Account for field length when scoring queries. Norms are useful for scoring, but also require a lot of disk space.',
```

Comment:
What does "Account for field length" mean here? What is field length and how does it affect the relevancy score?

The docs mention that fields used only for filtering or aggregations don't need norms. Do you think it's worth mentioning this in the UI?

Also, the docs seem to imply that norms can be disabled on an index but not re-enabled. Does this mean that this field's default value should be `true`?

Reply:
I believe it means as it sounds, the length of the actual field. More info here: https://www.elastic.co/guide/en/elasticsearch/guide/current/scoring-theory.html#field-norm

I do not think `norms` should be `true` by default. Since this is part of the index template flow, the mapping will be applied at index creation time, and the default `norms` value for `keyword` is `false`.

What about something like this? 
> Account for field length when scoring queries. Norms require significant memory and are not necessary for fields that are used solely for filtering or aggregations.